easy-install.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+MDBPW="mypass"
+ADMINPW="mypass"
+SITENAME="site1.local"
+BRANCH="develop"
+FRAPPEBRANCH="develop"
+while getopts ":m:a:s:b:f" OPTION; do
+
+  case "$OPTION" in
+    m)
+      MDBPW="$OPTARG"
+      ;;
+    a)
+      ADMINPW="$OPTARG"
+      ;;
+    s)
+      SITENAME="$OPTARG"
+      ;;
+    b)
+      BRANCH="$OPTARG"
+      ;;
+    f)
+      FRAPPEBRANCH="$OPTARG"
+      ;;
+    *)
+      echo "Usage: $(basename $0) [-a admin password] [-m mariadb password] [-s sitename] [-b ERPNext branch] [-f Frappe branch]"
+      exit 1
+      ;;
+  esac
+done
+
+useradd -m -s /usr/bin/bash frappe
+echo 'frappe ALL=(ALL) NOPASSWD: ALL' | sudo tee -a /etc/sudoers
+sudo -i -u frappe bash << EOF
+source ~/.profile
+
+# Install docker if it doesn't exist
+if ! command -v docker &> /dev/null
+then
+    wget https://get.docker.com/ -O install-docker.sh
+    sh install-docker.sh
+fi
+# Install Nix if it doesn't exist
+if ! command -v nix-env &> /dev/null
+then
+    wget https://nixos.org/nix/install -O install-nix.sh
+    sh install-nix.sh --daemon
+fi
+
+# Restart Shell
+
+exec $SHELL
+
+
+# Install the required packages with Nix
+nix-env -iA \
+    nixpkgs.nodejs-14_x \
+    nixpkgs.yarn \
+    nixpkgs.redis \
+    nixpkgs.python39Full \
+    nixpkgs.python39Packages.pip \
+    nixpkgs.python39Packages.setuptools \
+    nixpkgs.python27 \
+    nixpkgs.nginx \
+    nixpkgs.gcc \
+    nixpkgs.gnumake \
+    nixpkgs.mariadb-client \
+
+
+# Install bench
+echo "Installing Bench"
+sudo apt install -y python3-pip fontconfig
+sudo pip install frappe-bench 
+
+
+# Export Path if non-existent
+
+export PATH="$HOME/.local/bin:$PATH"
+
+# Install base fonts for wkhtmltopdf
+
+git clone --progress --depth 1 https://github.com/frappe/fonts.git /tmp/fonts 
+
+sudo rm -rf /etc/fonts && mv /tmp/fonts/etc_fonts /etc/fonts
+sudo rm -rf /usr/share/fonts && mv /tmp/fonts/usr_share_fonts /usr/share/fonts
```

Comment:
Nuking someone's font dir is probably not the best idea

Reply:
🙈 . Now uses apt to install required fonts for wkhtmltopdf. 

Ref: https://github.com/frappe/bench/blob/45267fc8979fe9bb7819a770df97482d87732577/bench/playbooks/roles/wkhtmltopdf/tasks/main.yml#L12-L18

Reply:
Can you confirm we're installing all fonts from https://github.com/frappe/fonts? I was thinking we should use something like https://github.com/alexmyczko/fnt to manage fonts for different users / requirements / setups 🤔